<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="LAPTOP-VSBTU4D1" failures="1" tests="1" name="reportingmethods.TestRunnerAddReportingMethod" time="30.195" errors="0" timestamp="2020-12-24T10:48:20 IST" skipped="0">
  <testcase classname="reportingmethods.TestRunnerAddReportingMethod" name="runScenario" time="30.195">
    <failure message="successful message is shown for adding a new reporting method" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: successful message is shown for adding a new reporting method
at org.junit.Assert.fail(Assert.java:89)
at reportingmethods.StepDefAddReportingMethod.a_new_reporting_method_is_added_as_message_is_displayed(StepDefAddReportingMethod.java:119)
at âœ½.a new Reporting Method is added as message is displayed(classpath:reportingmethods/addreportingmethod.feature:13)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <system-out/>
</testsuite> <!-- reportingmethods.TestRunnerAddReportingMethod -->
